np15.py

```diff
@@ -0,0 +1,14 @@
+import numpy as np
+
+
+def create_array():
+    """
+    Function that create a three-dimension array with shape (300,400,5) and
+    fill the array elements with value between 0 and 255
+    """
+    array = np.random.randint(0, 256, (300, 400, 5))
+    print(array)
+
+
+if __name__ == '__main__':
+    create_array()
```

Comment:
This should not be here.
Don’t use git add -a ever, use git add -p
https://git-scm.com/docs/git-add